tutorials/preprocessing/plot_15_handling_bad_channels.py

```diff
@@ -0,0 +1,254 @@
+# -*- coding: utf-8 -*-
+"""
+.. _tut-bad-channels:
+
+Interpolating bad channels
+==========================
+
+This tutorial covers manual marking of bad channels and reconstructing bad
+channels based on good signals at other sensors.
+
+.. contents:: Page contents
+   :local:
+   :depth: 2
+
+As usual we'll start by importing the modules we need, and loading some example
+data:
+"""
+
+import os
+import mne
+
+sample_data_folder = mne.datasets.sample.data_path()
+sample_data_raw_file = os.path.join(sample_data_folder, 'MEG', 'sample',
+                                    'sample_audvis_raw.fif')
+raw = mne.io.read_raw_fif(sample_data_raw_file, verbose=False)
+
+###############################################################################
+# Marking bad channels
+# ^^^^^^^^^^^^^^^^^^^^
+#
+# Sometimes individual channels malfunction and provide data that is too noisy
+# to be usable. MNE-Python makes it easy to remove bad channels from the
```

Comment:
It's probably a technical detail but something I hear often from purists is that the channels themselves are not necessarily bad but the data in the channels are bad. So perhaps you want to clarify at first that we use "bad channels" as a shorthand for channels containing artifacts.

Reply:
I tweaked the language a little, but I think "bad channels" is synonymous with "channels with bad data" for ~99% of users. Putting in your suggested caveat seems like overkill / distraction.